rotkehlchen/globaldb/updates/types.py

```diff
@@ -0,0 +1,36 @@
+from enum import Enum, auto
+from typing import NamedTuple
+
+from rotkehlchen.assets.types import AssetType
```

Comment:
Can you rename the directory from `/updates` to `asset_updates`? This will make it clear what the difference to the `upgrades/` directory is.